pytest_dotenv/plugin.py

```diff
@@ -15,4 +15,4 @@ def pytest_addoption(parser):
 @pytest.hookimpl(tryfirst=True)
 def pytest_load_initial_conftests(args, early_config, parser):
     for file in early_config.getini("env_files"):
-        load_dotenv(file)
+        load_dotenv(file, override=True)
```

Comment:
I'd prefer to make this optional:

1) add a new entry for the parser (like: https://github.com/quiqua/pytest-dotenv/blob/6f534861ef6da9bcbcfa689dfe09e1efac9b01f9/pytest_dotenv/plugin.py#L9-L12)

2) pass flag in https://github.com/quiqua/pytest-dotenv/blob/6f534861ef6da9bcbcfa689dfe09e1efac9b01f9/pytest_dotenv/plugin.py#L18